### PR TITLE
fix(dashboard): prevent audit rows overlapping pager

### PR DESCRIPTION
**Pull request by LABVETNEB**
## Summary

Fixes the CURRENT admin Audit layout defect discovered while executing A03.

At 1280x720, the populated audit table overflowed into the pager and intercepted pointer events from the real `Página siguiente` control.

The root cause was a self-referential adaptive measurement: Audit measured the content-sized table instead of the flex-assigned rows region, leaving desktop pagination pinned at 9 rows even where those rows did not fit.

The final review hardening also accounts for the measured region's 8px top padding while preserving the existing 6px safety clearance. This closes the intermediate-height boundary condition without reducing the established 9-row contract at canonical 1366x768 and 1440x900 viewports.

## Scope

- [ ] Backend runtime
- [x] Frontend runtime
- [ ] Tests
- [ ] Workflows/CI
- [ ] Migrations/schema
- [ ] Docs
- [ ] Dependencies
- [ ] Scripts/tooling
- [ ] Repository configuration
- [ ] Other
- [ ] Mixed-scope exception

Primary scope is frontend runtime. Tests and A02 baseline changes are supporting validation/guard realignment for the same frontend defect.

Changes:

- measure the real flex-assigned Audit rows region
- keep 9 rows as the desktop cap rather than an unconditional minimum
- remove the obsolete content-sized table measurement ref
- account for 8px desktop top padding in adaptive row capacity while preserving the existing 6px safety gap
- cover 1280x720, boundary viewport 1280x736, 1366x768 and 1440x900
- realign exactly one legitimately affected A02 record on each captured platform

A02 maintenance realignment is limited to:

`admin-auditoria::w1280x720`

Changed metrics:

- `regions.collection.height: 364.5 -> 290.5`
- `derived.collectionCanvasHeight: 364.5 -> 290.5`

The other 12 `admin-auditoria` canonical viewports remained unchanged.

A02 realignment covers the same single record on both captured platforms (`records` Windows and `platformRecords.linux`); no tolerance, selector, measurement-semantics, schema, or global-metadata changes.

The final 14px desktop adaptive gap hardening produces zero additional A02 drift.

A03 remains unchanged and paused pending this corrective PR.

## Validation

- pager regression 1280x720 / 1280x736 / 1366x768 / 1440x900: PASSED 4/4
- boundary 1280x736: old calculation 8 rows, corrected calculation 7 rows
- minimum measured canonical footer clearance: 7.19px at 1366x768
- `admin-audit-enterprise-density`: PASSED 6/6
- A02 `admin-auditoria` across all 13 canonical viewports, capture OFF: PASSED with zero drift
- `pnpm --dir frontend lint`: PASSED
- `pnpm --dir frontend typecheck`: PASSED
- `pnpm --dir frontend build`: PASSED
- `pnpm security:public-surface`: PASSED
- `pnpm --dir frontend e2e:visual-contract`: PASSED 277/277
- `git diff --check`: PASSED
- local `e2e:full`: NOT_RUN by protocol

## Rollback

Rollback this corrective PR as one reviewed unit through the normal GitHub revert workflow.

The runtime measurement correction, Windows/Linux A02 maintenance records, and boundary regression coverage represent the same corrected geometry and should not be partially reverted independently.

A partial rollback could restore either the original pager overlap or an A02 baseline inconsistent with runtime.

### CODE REVIEW

File: frontend/src/app/dashboard/admin/AdminAuditCard.tsx [hunk 1]
```diff
@@ -328,12 +332,17 @@ export function AdminAuditCard({
...
       />
 
-      <div className="min-h-0 flex-1 py-2">
+      {/* Measured rows region. It is the flex-allocated space between the
+          filter bar and the pager (`min-h-0 flex-1` keeps it shrinkable), so
+          its height is the space that really exists. The table wrapper inside
+          it is content-sized, which made the previous measurement
+          self-referential: nine rows measured nine rows' worth of height and
+          the fit never fell below nine, whatever the viewport. */}
+      <div ref={setDesktopBodyNode} className="min-h-0 flex-1 py-2">
```
[the remark this answers is not in the document]
Comment (LABVETNEB) [reply frontend/src/app/dashboard/admin/AdminAuditCard.tsx:360]: Addressed in d82d0ff6.

The measured desktop region includes 8px top padding. The adaptive calculation now preserves the existing 6px safety clearance and additionally accounts for that 8px consumed before the table, for an effective desktop gap of 14px.

For N rows:

H >= 32 + 14 + 37N

while the table ends at:

8 + 32 + 37N <= H - 6

so at least 6px clearance is guaranteed at every row-count boundary.

A 1280x736 boundary regression was added: the previous calculation selected 8 rows while the corrected calculation selects 7. Canonical 1280x720 remains 7 rows, 1366x768 remains 9, 1440x900 remains 9, and 1024x768 remains 9.

A02 admin-auditoria remains 13/13 PASSED with zero drift and visual-contract passes 277/277.
